android/capacitor/src/main/java/com/getcapacitor/Splash.java

```diff
@@ -47,7 +48,14 @@ private static void buildViews(Context c) {
       String splashResourceName = Config.getString(CONFIG_KEY_PREFIX + "androidSplashResourceName", "splash");
 
       int splashId = c.getResources().getIdentifier(splashResourceName, "drawable", c.getPackageName());
-      Drawable splash = c.getResources().getDrawable(splashId, c.getTheme());
+
```

Comment:
I guess this is to prevent some crash, but if the splash is not found it still crash on line 267
Can be prevented by changing to 

```
} catch (IllegalStateException | IllegalArgumentException ex) {
  Log.d(LogUtils.getCoreTag(), "Could not add splash view");
  return;
}
```